### PR TITLE
[MM-56019] Move the deactivate account section to the last, and change the button to danger

### PR DESCRIPTION
**Pull request by Syed-Ali-Abbas-Zaidi**
#### Summary

Move the deactivate account section to the last, and change the button to danger

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/25675
Jira https://mattermost.atlassian.net/browse/MM-56019

#### Screenshots

|  before  |  after  |
|----|----|
| image | Screenshot 2023-12-12 at 3 19 29 PM |

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by asaadmahmood — APPROVED** [review]
LGTM

File: webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx [hunk 1]
```diff
@@ -880,6 +881,11 @@ export default class AdvancedSettingsDisplay extends React.PureComponent<Props,
             );
         }
 
+        let deactivateAccountSectionDivider = null;
+        if (deactivateAccountSection) {
+            deactivateAccountSectionDivider = <div className='divider-light'/>;
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx:886]: so why do we need this divider?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx (hunk 1 above)
Comment (Syed-Ali-Abbas-Zaidi) [reply webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx]: You were right. We didn't need a divider, I have updated the code.

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — CHANGES_REQUESTED** [review]
working as expected

**Review by yasserfaraazkhan — APPROVED** [review]

**Review by harshilsharma63 — APPROVED** [review]
Looks good!

**Pull request merged**